addon/helpers/array.js

```diff
@@ -3,13 +3,12 @@
  * Temporary fix until the following bug is resolved: https://github.com/emberjs/ember.js/issues/14264
  */
 import Ember from 'ember'
-const {A, Helper} = Ember
+const {A: emberA, Helper} = Ember
```

Comment:
I know we'll be changing this at some point in the future (2.16+) but `emberArray` feels better than `emberA`.

Reply:
done